istio-custom-ingressgateways/templates/deployment.yaml

```
@@ -0,0 +1,87 @@
+{{- range $spec := .Values.global.customGateways }}
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ $spec.name }}
+  namespace: {{ $.Release.Namespace }}
+  labels:
+    {{- range $key, $val := $spec.labels }}
+    {{ $key }}: {{ $val }}
+    {{- end }}
+spec:
+  replicas: {{ $spec.replicaCount }}
+  template:
+    metadata:
+      labels:
+        {{- range $key, $val := $spec.labels }}
+        {{ $key }}: {{ $val }}
+        {{- end }}
+      annotations:
+        sidecar.istio.io/inject: "false"
+        scheduler.alpha.kubernetes.io/critical-pod: ""
+    spec:
+      serviceAccountName: {{ $spec.name }}-service-account
+      containers:
+        - name: istio-proxy
+          image: "gcr.io/istio-release/proxyv2:release-1.0-latest-daily"
+          imagePullPolicy: IfNotPresent
+          ports:
+            {{- range $key, $val := $spec.ports }}
+            - containerPort: {{ $val.port }}
+            {{- end }}
+          args:
+          - proxy
+          - router
+          - -v
+          - "2"
+          - --discoveryRefreshDelay
+          - '1s' #discoveryRefreshDelay
+          - --drainDuration
+          - '45s' #drainDuration
+          - --parentShutdownDuration
+          - '1m0s' #parentShutdownDuration
+          - --connectTimeout
+          - '10s' #connectTimeout
+          - --serviceCluster
+          - {{ $spec.name }}
+          - --zipkinAddress
+          - {{$.Values.global.tracingServiceHost }}:9411
```

It would be good to have consistency in the templating.
```suggestion
          - {{ $.Values.global.tracingServiceHost }}:9411
```
This is happening in other places as well.